documentation/endpoints/kudos.md

```diff
@@ -20,16 +21,75 @@ GET https://www.dosomething.org/api/v1/kudos
 
 ### Optional query parameters
 
+**NOTE: The following query parameters no longer work with this endpoint and will soon be deprecated or refactored!**
+
 - **reportbackitems** _(string|csv)_
```

Comment:
A little concerned about this one... @aaronschachter do you recall if we use this anywhere? I wanna say it was added when Kudos was originally created for the mobile app, but then was scrapped, so not 100% sure.